src/Squidex/app/shared/state/contents.state.ts

```diff
@@ -263,7 +263,7 @@ export abstract class ContentsStateBase extends State<Snapshot> {
 
                 this.replaceContent(updated, content.version);
             }),
-            shareSubscribed(this.dialogs));
+            shareSubscribed(this.dialogs, {silent: true}));
```

Comment:
Can you add whitespaces here after the `{` and before `}`... just `{ silent: true }`